blaze-client/src/main/scala/org/http4s/blaze/client/Http1Connection.scala

```diff
@@ -214,7 +216,10 @@ private final class Http1Connection[F[_]](
             }
 
           val idleTimeoutF: F[TimeoutException] = idleTimeoutStage match {
-            case Some(stage) => F.async_[TimeoutException](stage.setTimeout)
+            case Some(stage) =>
+              F.async[TimeoutException] { cb =>
+                F.delay(stage.setTimeout(cb)).as(noopCancel)
```

Comment:
Hm, is `cancelTimeout()` not the right thing to do here?
https://github.com/http4s/blaze/blob/8cf873e228c89b8ed836d7c635ab8142222ebc85/blaze-core/src/main/scala/org/http4s/blaze/core/IdleTimeoutStage.scala#L134

Reply:
That seems legit.  In practice, I think the stage is going to shutdown before any of this matters.